### ISSUE TITLE
Can this tool install with bower?

### ISSUE THREAD

**Issue by vincent-ng** [blackat/ui-navbar#2]
It really a cool navbar directive. I think it's preferable to install with bower

**Comment by blackat** [comment]
Hi,
I am glad you like it, I am agree that by Bower it should be better, I will have a look at the process to make available via bower.

**Comment by blackat** [comment]
@vincentngthu ui-navbar is available via Bower

```
bower info ui-navbar
```

thanks for the suggestion :wink:

**Comment by vincent-ng** [comment]
Thank your for add it to bower.

Two more suggest:
- 1. When I use "bower install ui-navbar --save", I found bower.json become

```
"dependencies": {
    "ui-navbar": "*"
}
```

It would be nice if you add a "git tag 0.2.4" to the commit.
So that the dependencies would become.

```
"dependencies": {
    "ui-navbar": "~0.2.4"
}
```

Better for versioning.
- 2. In the bower.json config, I found:

```
"main": "dist/ui-navbar.js"
```

But the release code is located at "release/js/ui-navbar.js".
It is better to have a correct main entry so that some other automatic tools can parse your bower.json.

Thx
